dts/bindings/led_strip/worldsemi,ws2812-gpio.yaml

```diff
@@ -12,3 +12,24 @@ description: |
 compatible: "worldsemi,ws2812-gpio"
 
 include: [base.yaml, ws2812-gpio.yaml]
+
+properties:
+  delay-t1h:
+    type: int
+    default: 12
```

Comment:
default values need justifications https://docs.zephyrproject.org/latest/build/dts/bindings-upstream.html#rules-for-default-values

Reply:
The updated pull request doesn't require changes to the device tree any more.